copier/copier.go

```diff
@@ -41,6 +41,74 @@ func init() {
 	reexec.Register(copierCommand, copierMain)
 }
 
+// extendedGlob calls filepath.Glob() on the passed-in patterns.  If there is a
+// "**" component in the pattern, filepath.Glob() will be called with the "**"
+// replaced with all of the subdirectories under that point, and the results
+// will be concatenated.
+func extendedGlob(pattern string) (matches []string, err error) {
+	subdirectories := func(dir string) []string {
+		var subdirectories []string
```

Comment:
This is difficult to read, since the function and the variable have the same name.  Would have been better if they were different.